examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,202 @@
+            let cameraTravel = [];
+
+            let isFirstPickedPoint = true;
+            let travelOn = 0;
+
+            // Add a pointer at a given position.
+            function addMeshToScene(position) {
+                // Creation of the new mesh :
+                let mesh = new itowns.THREE.AxesHelper(50000);
+                mesh.position.copy(position.as(view.referenceCrs));
+                mesh.lookAt(position.geodesicNormal);
+                mesh.rotateY(Math.PI);
+                mesh.updateMatrixWorld();
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearMeshes() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Utility method to compute the positive mod.
+            Number.prototype.mod = function(n) {
+                const m = ((this % n) + n) % n;
+                return m < 0 ? m + Math.abs(n) : m;
+            }
+
+            // Listens for user input :
+            window.addEventListener('keypress', async function (event) {
+
+                // If user presses `s` (for 'save'), saves the current camera position, heading, tilt...
+                if (event.key === 's') {
+                    // Get camera coordinates :
+                    let cameraCoord3D = new itowns.Coordinates(view.referenceCrs);
+                    cameraCoord3D.setFromVector3(view.camera.camera3D.position);
+
+                    // Saves camera current data :
+                    let pathStep = {
+                        coord: view.controls.getLookAtCoordinate().as(view.referenceCrs),
+                        coord3D: cameraCoord3D,
+                        range: view.controls.getRange(),
+                        heading: view.controls.getHeading(),
+                        tilt: view.controls.getTilt(),
+                    };
+                    if (isFirstPickedPoint) {
+                        pathStep.time = 0;
+                        // Instant move to the first position of the travel :
+                        isFirstPickedPoint = false;
+                    } else {
+                        // The movement duration between two positions is proportional to the distance between them.
+                        // This gives a feeling that the camera is moving at a constant speed.
+                        let ellipsoid = new itowns.Ellipsoid();
+                        let currentCartesian = ellipsoid.cartographicToCartesian(cameraCoord3D.as('EPSG:4326'));
+                        let previousCartesian = ellipsoid.cartographicToCartesian(
+                            cameraTravel[cameraTravel.length - 1].coord3D.as('EPSG:4326'),
+                        );
+                        pathStep.time = Math.sqrt(
```

Comment:
Could you use THREE.Vector3 Method to compute length?

Reply:
I now use `THREE.Vector3` for distance computing.